2018-08-23 15:41:56.134 [info] <0.42.0> Application lager started on node nonode@nohost
2018-08-23 15:41:56.137 [info] <0.42.0> Application xmerl started on node nonode@nohost
2018-08-23 15:41:56.139 [info] <0.42.0> Application jsx started on node nonode@nohost
2018-08-23 15:41:56.142 [info] <0.42.0> Application ranch started on node nonode@nohost
2018-08-23 15:41:56.143 [info] <0.42.0> Application ranch_proxy_protocol started on node nonode@nohost
2018-08-23 15:41:56.143 [info] <0.42.0> Application recon started on node nonode@nohost
2018-08-23 15:41:56.144 [info] <0.42.0> Application rabbit_common started on node nonode@nohost
2018-08-23 15:41:56.149 [info] <0.42.0> Application amqp_client started on node nonode@nohost
2018-08-23 15:41:56.149 [info] <0.42.0> Application amqp started on node nonode@nohost
2018-08-23 15:41:57.095 [warning] <0.318.0> lager_error_logger_h dropped 4 messages in the last second that exceeded the limit of 50 messages/sec
2018-08-23 15:53:23.844 [error] <0.950.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':start_link({error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}) at <0.955.0> exit with reason normal in context child_terminated
2018-08-23 15:53:23.873 [info] <0.33.0> Application lager started on node nonode@nohost
2018-08-23 15:53:23.876 [info] <0.33.0> Application xmerl started on node nonode@nohost
2018-08-23 15:53:23.878 [info] <0.33.0> Application jsx started on node nonode@nohost
2018-08-23 15:53:23.882 [info] <0.33.0> Application ranch started on node nonode@nohost
2018-08-23 15:53:23.883 [info] <0.33.0> Application ranch_proxy_protocol started on node nonode@nohost
2018-08-23 15:53:23.883 [info] <0.33.0> Application recon started on node nonode@nohost
2018-08-23 15:53:23.884 [info] <0.33.0> Application rabbit_common started on node nonode@nohost
2018-08-23 15:53:23.891 [info] <0.33.0> Application amqp_client started on node nonode@nohost
2018-08-23 15:53:23.891 [info] <0.33.0> Application amqp started on node nonode@nohost
2018-08-23 15:53:23.906 [error] <0.1007.0>@Elixir.GenAMQP.Conn:init:78 CRASH REPORT Process <0.1007.0> with 0 neighbours crashed with reason: no match of right hand value {error,econnrefused} in 'Elixir.GenAMQP.Conn':init/1 line 78
2018-08-23 15:53:23.906 [error] <0.1006.0>@Elixir.GenAMQP.Conn:init:78 Supervisor 'Elixir.StaticConnSup' had child 'Elixir.GenAMQP.Conn' started with 'Elixir.GenAMQP.Conn':start_link(<<"amqp://guest:guest@localhost">>, 'Elixir.ConnHub') at undefined exit with reason no match of right hand value {error,econnrefused} in 'Elixir.GenAMQP.Conn':init/1 line 78 in context start_error
2018-08-23 15:53:23.906 [error] <0.1005.0> Supervisor 'Elixir.GenAMQP.AppSupervisor' had child 'Elixir.StaticConnSup' started with 'Elixir.GenAMQP.ConnSupervisor':start_link('Elixir.StaticConnSup', 'Elixir.ConnHub', <<"amqp://guest:guest@localhost">>) at undefined exit with reason {shutdown,{failed_to_start_child,'Elixir.GenAMQP.Conn',{{badmatch,{error,econnrefused}},[{'Elixir.GenAMQP.Conn',init,1,[{file,"lib/gen_amqp/conn.ex"},{line,78}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,365}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,333}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]}}} in context start_error
2018-08-23 15:53:23.906 [error] <0.1003.0> CRASH REPORT Process <0.1003.0> with 0 neighbours exited with reason: {{shutdown,{failed_to_start_child,'Elixir.StaticConnSup',{shutdown,{failed_to_start_child,'Elixir.GenAMQP.Conn',{{badmatch,{error,econnrefused}},[{'Elixir.GenAMQP.Conn',init,1,[{file,"lib/gen_amqp/conn.ex"},{line,78}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,365}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,333}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]}}}}},{'Elixir.DemoApp',start,[normal,[]]}} in application_master:init/4 line 134
2018-08-23 15:53:23.975 [info] <0.33.0> Application gen_amqp exited with reason: {{shutdown,{failed_to_start_child,'Elixir.StaticConnSup',{shutdown,{failed_to_start_child,'Elixir.GenAMQP.Conn',{{badmatch,{error,econnrefused}},[{'Elixir.GenAMQP.Conn',init,1,[{file,"lib/gen_amqp/conn.ex"},{line,78}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,365}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,333}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]}}}}},{'Elixir.DemoApp',start,[normal,[]]}}
2018-08-23 15:54:58.463 [error] <0.171.0> Supervisor 'Elixir.Logger.Supervisor' had child 'Elixir.Logger.ErrorHandler' started with 'Elixir.Logger.Watcher':start_link({error_logger,'Elixir.Logger.ErrorHandler',{true,false,500}}) at <0.176.0> exit with reason normal in context child_terminated
2018-08-23 15:54:58.502 [info] <0.33.0> Application lager started on node nonode@nohost
2018-08-23 15:54:58.505 [info] <0.33.0> Application xmerl started on node nonode@nohost
2018-08-23 15:54:58.507 [info] <0.33.0> Application jsx started on node nonode@nohost
2018-08-23 15:54:58.510 [info] <0.33.0> Application ranch started on node nonode@nohost
2018-08-23 15:54:58.512 [info] <0.33.0> Application ranch_proxy_protocol started on node nonode@nohost
2018-08-23 15:54:58.513 [info] <0.33.0> Application recon started on node nonode@nohost
2018-08-23 15:54:58.513 [info] <0.33.0> Application rabbit_common started on node nonode@nohost
2018-08-23 15:54:58.517 [info] <0.33.0> Application amqp_client started on node nonode@nohost
2018-08-23 15:54:58.517 [info] <0.33.0> Application amqp started on node nonode@nohost
2018-08-23 15:54:59.117 [warning] <0.32.0> lager_error_logger_h dropped 47 messages in the last second that exceeded the limit of 50 messages/sec
